src/components/number/number.jsx

```diff
@@ -75,16 +76,13 @@ export default function Number({
   const tickDecimals = getTickDecimals(value, ticks);
   const minimumFractionDigits = getFractionDigits(tickDecimals, valueMinDecimals, valueDecimals);
   const maximumFractionDigits = getFractionDigits(tickDecimals, valueMaxDecimals, valueDecimals);
+  const formattedNumber = formatNumber(value, { minimumFractionDigits, maximumFractionDigits });
 
   return (
-    <span {...rest} className={classes} style={styles}>
+    <span {...rest} className={classes} style={styles} title={formattedNumber}>
```

Comment:
Place title attribute to the far left so that the resulting row is: 
`<span title={formattedNumber} {...rest} className={classes} style={styles}>`

This will allow us to overwrite the title attribute if we see fit.